channel/consensus_channel/serde_test.go

```diff
@@ -12,29 +12,26 @@ import (
 )
 
 func TestSerde(t *testing.T) {
-	// TODO unskip this test when we have solved the issue for persisting big.Ints
-	// https://github.com/statechannels/go-nitro/issues/439
-	t.Skip()
+
 	someGuarantee := Guarantee{
-		amount: *big.NewInt(1),
+		amount: big.NewInt(1),
 		left:   testdata.Actors.Alice.Destination(),
 		right:  testdata.Actors.Alice.Destination(),
 		target: types.Destination{99},
 	}
 	someOutcome := makeOutcome(
-		Balance{testdata.Actors.Alice.Destination(), *big.NewInt(2)},
-		Balance{testdata.Actors.Bob.Destination(), *big.NewInt(7)},
+		Balance{testdata.Actors.Alice.Destination(), big.NewInt(2)},
+		Balance{testdata.Actors.Bob.Destination(), big.NewInt(7)},
 		someGuarantee)
 
 	t.Run("Guarantee", func(t *testing.T) {
 		got, err := json.Marshal(someGuarantee)
 		if err != nil {
 			t.Fatal(err)
 		}
-		// TODO: this expectation is not right and currently has some placeholders  / gaps
-		want := `{"Amount":{SOMETHING!},"Target":"0x6300000000000000000000000000000000000000000000000000000000000000","Left":"0x000000000000000000000000aaa6628ec44a8a742987ef3a114ddfe2d4f7adce","Right":"0x000000000000000000000000aaa6628ec44a8a742987ef3a114ddfe2d4f7adce"}`
+		want := `{"Amount":1,"Target":"0x6300000000000000000000000000000000000000000000000000000000000000","Left":"0x000000000000000000000000aaa6628ec44a8a742987ef3a114ddfe2d4f7adce","Right":"0x000000000000000000000000aaa6628ec44a8a742987ef3a114ddfe2d4f7adce"}`
```

Comment:
Interesting that `*big.Int` gets marshalled to a plain old number. We should think about whether that may cause us problems when the numbers are big! It may only be an issue when wanting to use the JSON in (say) javascript.